src/commands/changelog.ts

```diff
@@ -4,10 +4,15 @@ exports.run = async (client, message) => {
       embed: {
         author: { name: client.user.username, icon_url: client.user.avatarURL },
         color: 0x1167b1,
-        title: "**V1.4.0**",
+        title: "**V1.4.1**",
         description:
           "A **lot** has changed in 1.4, and the team has almost entirely changed, so this isn't an exhaustive list.",
         fields: [
+          {
+            name: "1.4.1 changes",
+            value: 
+              "- Added the `fact` command! Use this command to find out some fun facts about Cytrus-RE.\n- Added embeds to `c.math` and `c.colourutil` (when you haven't specified an action).\n- Fixed a few bugs (specifically, the version number was outdated in a few places + some code fixes)."
+          },
```

Comment:
this should be at the bottom

Reply:
no, since they're the latest changes and because it's a hotfix they should really be at the top imo

Reply:
nah i think it should be at the bottom in a "hotfixes" section or something like that.

Reply:
I'd expect it to be at the top if I were a user because, well, it's the newest

Reply:
well atleast put in a section for hotfixes because we're probably gonna work a lot on 1.4